pkg/tls/marshal.go

```diff
@@ -29,6 +29,25 @@ func CertAsBytes(cert *x509.Certificate) (b []byte, err error) {
 	return buf.Bytes(), nil
 }
 
+func CertChainAsBytes(certs []*x509.Certificate) (b []byte, err error) {
```

Comment:
investigate if can be collapsed, one route is a smaller wrapper function in the template FuncMap